mesonbuild/compilers/d.py

```diff
@@ -27,6 +27,21 @@
     CompilerArgs,
 )
 
+d_feature_args = {
+                  'gcc':  {'unittest': '-funittest',
```

Comment:
__[flake8]__
* [E126] continuation line over-indented for hanging indent

<!-- Comment by [SideCI](https://sideci.com) -->